server/src/completionUtil.ts

```diff
@@ -63,119 +67,219 @@ export const blockAttributes: CompletionItem[] = [
-    documentation:
-      'Defines the primary key. There must be exactly one field @id or block @id',
+    documentation: {
+      kind: MarkupKind.Markdown,
+      value: [
+        '```prisma',
+        '@id',
+        '```',
+        '___',
+        'Defines a single-field ID on the model.',
+      ].join('\n'),
+    },
   },
   {
     label: '@unique',
     kind: CompletionItemKind.Property,
-    detail: '@unique',
-    documentation: 'Defines the unique constraint.',
+    documentation: {
+      kind: MarkupKind.Markdown,
+      value: [
+        '```prisma',
+        '@unique',
+        '```',
+        '___',
+        'Defines a unique constraint for this field.',
+      ].join('\n'),
+    },
   },
   {
     label: '@map()',
     kind: CompletionItemKind.Property,
-    detail: '@map(_ name: String)',
     insertTextFormat: 2,
     insertText: '@map($0)',
-    documentation: 'Defines the raw column name the field is mapped to.',
+    documentation: {
+      kind: MarkupKind.Markdown,
+      value: [
+        '```prisma',
+        '@map(_ name: String)',
+        '```',
+        '___',
+        'Maps a field name from the Prisma schema to a different column name.',
+        '',
+        '_@param_ `name` The name of the target database column',
+      ].join('\n'),
+    },
   },
   {
     label: '@default()',
     kind: CompletionItemKind.Property,
-    detail: '@default(_ expr: Expr)',
     insertTextFormat: 2,
     insertText: '@default(0)',
-    documentation: 'Specifies a default value if null is provided.',
+    documentation: {
+      kind: MarkupKind.Markdown,
+      value: [
+        '```prisma',
+        '@default(_ expression: Expression)',
+        '```',
+        '___',
+        'Defines a default value for this field. `@default` takes an expression as an argument.',
+        '',
+        '_@param_ `expression` An expression (e.g. `5`, `true`, `now()`)',
+      ].join('\n'),
+    },
   },
   {
     label: '@relation()',
     kind: CompletionItemKind.Property,
     insertTextFormat: 2,
     insertText: '@relation($0)',
-    detail:
-      '@relation(_ name?: String, references?: Identifier[], onDelete?: CascadeEnum)\nArguments:\n•name: (optional, except when required for disambiguation) defines the name of the relationship. The name of the relation needs to be explicitly given to resolve amibiguities when the model contains two or more fields that refer to the same model (another model or itself).\n•references: (optional) list of field names to reference',
+    documentation: {
+      kind: MarkupKind.Markdown,
+      value: [
+        '```prisma',
+        '@relation(_ name: String?, fields: FieldReference[]?, references: FieldReference[]?)',
+        '```',
+        '___',
+        'Defines meta information about the relation. [Learn more](https://www.prisma.io/docs/reference/tools-and-interfaces/prisma-schema/relations#the-relation-attribute).',
+        '',
+        '_@param_ `name` A name of field references',
```

Comment:
What does this mean?

Reply:
![doc_comments](https://user-images.githubusercontent.com/30771368/83530463-0d43b680-a4ec-11ea-9348-e8d7f131c1ef.png)
This would look like this.

Reply:
No, what are "field references"?

"The name of the field that is referenced by the relation" or something like that would have meaning to me.

Reply:
I only used the documentation from Prisma Docs to be consistent.